src/lightning_lite/connector.py

```diff
@@ -184,7 +184,8 @@ def _check_config_and_set_final_flags(
         if strategy is not None and strategy not in self._registered_strategies and not isinstance(strategy, Strategy):
             raise ValueError(
                 f"You selected an invalid strategy name: `strategy={strategy!r}`."
-                f" Available names are: {', '.join(self._registered_strategies)}."
+                " Example choices: ddp, ddp_spawn, deepspeed, dp, ..."
+                " Find a complete list of options in our documentation at https://lightning.ai"
```

Comment:
Don't we have a more specific link than the website root?

Reply:
Yes, I could point directly to the Lite page. Are we ok with the risk of it getting outdated?

Reply:
You are right, with all the docs changes going on, it's probably better not to do it yet

Reply:
Yes, let's take the risk.